scripts/build.js

```diff
@@ -10,6 +10,7 @@ const args = parseArgs(process.argv.slice(2), {
 (async () => {
   const { globby } = await import('globby');
   const destinationPath = 'dist';
+  const isProduction = process.env.NODE_ENV === 'production';
```

Comment:
I think we should do this "hack" only for storybook builds.

Reply:
I think it should work for every build except releases. If I add a condition for only storybook builds, it will not work well in the development environment. Any advice for this?

Reply:
In development environment (like in playground, or linking to another project to test etc) we should never need that to use this class name. Class name express itself: `__ONLY_FOR_STORYBOOK_DEMONSTRATION_HOVER__` 😊

Reply:
There are some other works on storybook flow (#203). Let's fix this detail there. I think we can go with this.